components/mailchimp/mailchimp.app.js

```diff
@@ -0,0 +1,681 @@
+const mailchimp = require("@mailchimp/mailchimp_marketing");
+const retry = require("async-retry");
+module.exports = {
```

Comment:
I suggest we change `list-event-types.js` to `constants.mjs` and list some additional values there including a default PAGE_SIZE of 1000, since I see that being used in several places.
```suggestion
const retry = require("async-retry");
const constants = require("./sources/constants.mjs");
// or import constants from "./sources/constants.mjs";

module.exports = {
```

```
//constants.mjs

export default {
  PAGE_SIZE: 1000,
  CAMPAIGN_TYPES: [
    "regular",
    "plaintext",
    "absplit",
    "rss",
    "variate",
  ],
  CAMPAIGN_STATUSES: [
    "created",
    "sent",
  ],
  FILE_TYPES: [
    "image",
    "file",
    "all",
  ],
  SEGMENT_WATCH_TYPES: [
    "Created",
    "Updated",
  ],
  ORDER_HAS_OUTREACH_CHOICES: [
    "Yes",
    "No",
    "Both",
  ],
  EVENT_TYPES: [
    {
      label: "Subscribes",
      value: "subscribe",
    },
    {
      label: "Unsubscribes",
      value: "unsubscribe",
    },
    {
      label: "Profile Updates",
      value: "profile",
    },
    {
      label: "Cleaned Email",
      value: "cleaned",
    },
    {
      label: "Email Address Changes",
      value: "upemail",
    },
    {
      label: "Campaign sending status",
      value: "campaign",
    },
  ];
}
```